x-pack/plugins/ml/public/ui_actions/open_vis_in_ml_action.tsx

```diff
@@ -28,7 +28,7 @@ export function createVisToADJobAction(
     },
     getDisplayName: () =>
       i18n.translate('xpack.ml.actions.createADJobFromLens', {
-        defaultMessage: 'Create anomaly detection job',
+        defaultMessage: 'Detect anomaly',
```

Comment:
```suggestion
        defaultMessage: 'Detect anomalies',
```

Reply:
Updated copy in [2ecb655](https://github.com/elastic/kibana/pull/178596/commits/2ecb655c5e68d909a3cfa9c1c21c0ef5110b620a) 👍